crates/control-interface/src/types/link.rs

```diff
@@ -7,11 +7,16 @@ use crate::{
 };
 
 /// A link definition between a source and target component (component or provider) on a given
-/// interface. An [`InterfaceLinkDefinition`] connects one component's import to another
+/// interface.
+///
+/// An [`InterfaceLinkDefinition`] connects one component's import to another
 /// component's export, specifying the configuration each component needs in order to execute
 /// the request, and represents an operator's intent to allow the source to invoke the target.
+///
+/// This InterfaceLinkDefinition is *distinct* from the one in `wasmcloud_core`, in that it is
```

Comment:
nit: changed name

Reply:
Ah thanks this should just be "link definition"